data/countries/SI.yaml

```diff
@@ -132,428 +135,7 @@ holidays:
           - from: 1991
       12-26:
         name:
-          sl: dan samostojnosti in enotnosti
+          sl: Dan samostojnosti in enotnosti
           en: Independence and Unity Day
     # regions:
-    # '001':
```

Comment:
Hi, can we keep these regions in here? Just in case that they are needed at a later time?

Reply:
Hi!

You are right, makes more sense to keep them. They've been talking about regions on and off for the last 20 years.

I'll revert this part.

Reply:
Done.